templates/openstack/consulting.html

```diff
@@ -341,7 +341,7 @@ <h3 class="p-matrix__title">Local regulatory compliance</h3>
           <div class="p-matrix__content">
             <h3 class="p-matrix__title">Interoperability</h3>
             <p class="p-matrix__desc">
-              Hardware specifications, software-defined networking (SDN) controllers, storage platforms. We provide you with the flexibility to integrate your desired functionality. We help you choose what fits your requirements to provide the best value for money.
+              We provide you with the flexibility to integrate your desired functionality &ndash; from Hardware specifications, software-defined networking (SDN) controllers, to storage platforms. We help you choose what fits your requirements provide you with the best value for money.
```

Comment:
```suggestion
              We provide you with the flexibility to integrate your desired functionality &ndash; from hardware specifications, software-defined networking (SDN) controllers, to storage platforms. We help you choose what fits your requirements provide you with the best value for money.
Comment
```
This capital H seems like it should be lowercased now since it is no longer the start of the sentence

Reply:
Updated on doc as well, thanks!